gapic/src/main/com/google/gapid/settings.proto

```diff
@@ -114,6 +114,7 @@ message Trace {
   string device_serial = 1;
   string device_name = 2;
   string type = 3;
+  bool default_no_angle = 21;
```

Comment:
Can we add this to the end instead of middle.

Reply:
It makes more logical sense to be here as it goes along with the type. Why add it to the end?

Reply:
Because there is a number `21` next to it. When someone needs to add another type in that proto, they shouldn't be looking at all the proto to find highest number. if that really needs to be next to it, then we should order the numbers again.

Protobuffers fundamentally are a backward compatible serialisation API. That is the first and foremost purpose of  protobuffers(or any such APIs). So the highest priority in using them and keeping things in order should be based on that. 

Luckily we don't have backward compatibility issues so that we can avoid most of the compatibility handling but we can at least keep the numbers in order. Any other ordering, alphabetical, logical etc. comes after that.

Reply:
Again, this proto is serialized and stored on disk, so we actually do have to keep backwards compatibility.

Having the tag IDs get out-of-order is, well, just a part of protobufs. Take a look at https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto.

In my opinion, before you add/delete/modify a protobuf message, you better read the entire message, so you understand how your change fits in with what is already there. Some people like to add a comment at the beginning of the message with the next tag ID to use. That is a strategy we could adopt, if we really do get into trouble figuring out what the next ID should be. I don't know, IDEs help with that, so I'm not worried about this.